lib/FusionInventory/Agent/SNMP/MibSupport/Digi.pm

```diff
@@ -55,9 +58,28 @@ sub run {
             my $simcard = {
                 IMSI    => $sarianGPRS->{'21.0'}, # gprsIMSI
                 ICCID   => $sarianGPRS->{'20.0'}, # gprsICCID
-                #STATE   => $sarianGPRS->{'26.0'}, # gprsSIMStatus
             };
 
+            my $operator = $sarianGPRS->{'22.0'}; # gprsNetwork
+            if ($operator) {
+                my ($name, $mcc, $mnc) = $operator =~ /^(.*),\s*(\d{3})(\d+)$/ ;
+                $simcard->{OPERATOR_NAME} = $name;
+                if ($mcc) {
+                    $simcard->{OPERATOR_CODE} = "$mcc.$mnc" if $mnc;
+                    FusionInventory::Agent::Tools::Standards::MobileCountryCode->require();
+                    unless ($EVAL_ERROR) {
+                        FusionInventory::Agent::Tools::Standards::MobileCountryCode->use('getCountryMCC');
```

Comment:
you can use FusionInventory::Agent::Tools::Standards::MobileCountryCode->use() instead of ->require(), to import all exported functions at once, and get rid of this additional line.